### ISSUE TITLE
Add Battery Camera w/ RTSP support

### ISSUE THREAD

**Issue by Blueion76** [blakeblackshear/frigate#6211]
**Describe what you are trying to accomplish and why in non-technical terms**
I want to be able to add my Eufy battery cameras to Frigate and have them record whenever the stream goes online and stop whenever the stream goes offline.

**Describe the solution you'd like**
I'd like Frigate to have the option to poll a camera more often to check for the RTSP stream. I would also like Frigate to stop clips whenever an RTSP stream goes offline and start them whenever the stream goes online. 

**Describe alternatives you've considered**
I have attempted to solve this by limiting clip length but sometimes clips stop before the 60 seconds I have them set to. This results in clips becoming scattered throughout the day.

**Additional context**
In short, more configuration options and/or being able to bring a camera in Frigate online via Home Assistant would be nice. I'd also like to be able to have clips stop/end when the stream goes offline/online.

**Comment by NickM-27** [comment]
https://github.com/blakeblackshear/frigate/issues/1911 would likely cover this in general, but frigate is designed to work with high quality cameras that are available 24/7 constants. battery cameras will provide a stream only during motion / object detection which can mean frigate misses a lot of context for events
